third_party/terraform/resources/resource_compute_instance_group_manager.go.erb

```diff
@@ -16,6 +16,13 @@ import (
 	"google.golang.org/api/compute/v1"
 )
 
+const (
+	// Some fields are only supported by region IGM - these flags are used to
+	// manage non-shared fields in shared expanders/flatteners.
+	includeMultiZoneFields = true
```

Comment:
```suggestion
	includeRegionalOnlyFields = true
```

Given that there's a precedent for a "multi-zonal" cluster that's distinct from a regional cluster in GKE, I'm a little hesitant to call this `MultiZone` instead of `Regional` in case multi-zonal evolves as a distinct concept in IGM. (It's pretty unlikely, admittedly)

Reply:
can switch, i used multizonal because that's what the error message said and I had to do the "that means regional" switch in my head